### PR TITLE
Allow to override SnackBar style in Notification Component

### LINKED ISSUE #2400: Cannot override SnackBar style in Notification Component
Since Notification component is using Snackbar, that would be great to customize SnackBar style for example.

**What you were expecting:**
I want to customize `anchorOriginBottomCenter` css property from SnackBar : 
`<Notification classes={{ root: ..., anchorOriginBottomCenter: classes.anchorPosition }}`

**What happened instead:**
`<Notification />` ignore classes 

**Steps to reproduce:**
Create a notification component, try to modify SnackBar through the css api https://material-ui.com/api/snackbar/#css-api

### PR DESCRIPTION
**Pull request by rameshsyn**
Fix #2400

### CODE REVIEW

**Comment by fzaninotto** [comment]
I've rebased the next branch on master (it was lagging behind the latest minor). Could you please rebase your PR on the new next?

**Comment by rameshsyn** [comment]
@fzaninotto :+1:

**Comment by fzaninotto** [comment]
There is something wrong with your rebase, if includes commits that are already in the `next` branch.

**Comment by rameshsyn** [comment]
@fzaninotto Yeah, my rebase was on the wrong branch. Now, I pulled new `next` branch changes and add `Notification` component changes on it and force push.

File: packages/ra-ui-materialui/src/layout/Notification.js [hunk 1]
```diff
@@ -71,7 +71,12 @@ class Notification extends React.Component {
             hideNotification,
             ...rest
         } = this.props;
-
+        const {
+            warning,
```
[the remark this answers is not in the document]
Comment (fzaninotto) [reply packages/ra-ui-materialui/src/layout/Notification.js]: bump, please replace `classes.undo` by `undoClass` line 104

[the remark this answers is not in the document]
Comment (rameshsyn) [reply packages/ra-ui-materialui/src/layout/Notification.js]: Oh sure, thanks.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**